.github/workflows/release.yml

```diff
@@ -0,0 +1,126 @@
+name: Release build
+env:
+  TAG: "latest"
+  DOCKER_ORG: enmasse
+  KUBERNETES_NAMESPACE: "enmasse-infra"
+  SYSTEMTEST_PROFILE: smoke
+  REGISTRY_USER: "enmasse+ci"
+
+on:
+  push:
+    branches:
+      - master
+      - actions
+    tags:
+      - 'v*'
+
+jobs:
+  unit:
+    name: Unit tests
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11.0.2
+    
+    - name: Setup Ruby environment with ruby-build
+      uses: clupprich/ruby-build-action@v1.0.0
+
+    - name: Install dependencies
+      run: ./.github/scripts/install_dep.sh
+
+    - name: Run unit tests
+      run: make
+
+    - name: Collect logs
+      if: failure()
+      run: ./.github/scripts/collectLogs.sh
+
+    - name: Upload artifacts
+      if: failure()
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        name: unit-logs
+        path: artifacts
+
+
+  smoke:
+    name: Smoke tests
+    needs: unit
+    env:
+      DOCKER_REGISTRY: "quay.io"
+      TAG: "latest"
+      SYSTEMTEST_PROFILE: smoke
```

Comment:
No need to define this (it feels wrong to define this when test.sh tests both smoke tests and OLM